lib/raster3d/mask.c

```diff
@@ -306,12 +306,12 @@ void Rast3d_mask_tile(RASTER3D_Map * map, int tileIndex, void *tile, int type)
 	for (dy = y; dy < rows; dy++) {
 	    for (dx = x; dx < cols; dx++) {
 		RASTER3D_MASKNUM(map, dx, dy, dz, tile, type);
-		tile += length;
+		tile = (int*)tile + length;
```

Comment:
The changes here are not Cmake related and should be separated out

Reply:
Definitely not related to CMake, should separated into new PR. BTW, there are more casting issues in mask.c.